src/Entities/BaseProduct.php

```diff
@@ -0,0 +1,27 @@
+<?php
+
+namespace RobotStores\Entities;
+
+class BaseProduct
+{
+    protected int $id;
+    protected string $title;
+    protected float $price;
+    protected string $image;
+
+    public function getID():int {
```

Comment:
missing space before return type

Reply:
also drop the curly boi brace down onto the next line for method definitions, same as for classes